hexa/workspaces/models.py

```diff
@@ -68,7 +68,9 @@ def create_if_has_perm(
             )
 
         db_password = User.objects.make_random_password(length=16)
-        db_name = format_db_name(create_kwargs["slug"])
+        db_name = format_db_name(
```

Comment:
@cheikhgwane this will work but after giving it some thoughts, I'd rather use a random uuid4 (without dashes) as the database name and role name.

Reply:
Okay, all right.